modules/migrating-to-virt.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/getting_started/virt-getting-started.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="migrating-to-virt_{context}"]
+= Migrating to {VirtProductName}
+
+To migrate virtual machines from an external provider such as VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, or another {VirtProductName} cluster, use the Migration Toolkit for Virtualization (MTV). You can also migrate Open Virtual Appliance (OVA) files created by VMware vSphere.
+
+[NOTE]
+====
+Migration Toolkit for Virtualization is not part of {VirtProductName} and requires separate installation. Because of that, all links in this procedure lead outside of {VirtProductName} documentation.
+====
+
+.Prerequisites
+* The Migration Toolkit for Virtualization Operator link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv[is installed].
```

Comment:
I'm inclined to use an attribute for the MTV version as well; otherwise, we would need to manually update the version number embedded in these links for each release. You could alternatively use "latest" instead of 2.7, but please make sure that's supportable first.

The common-attributes file has these for MTC:

```
:mtc-version: 1.8
:mtc-version-z: 1.8.5
```